### PR TITLE
Add team annotation to Chart.yaml

### PR DESCRIPTION
**Pull request by rossf7**
Towards https://github.com/giantswarm/giantswarm/issues/14846

### CODE REVIEW

**Review by njuettner — APPROVED** [review]

File: helm/aws-operator/Chart.yaml [hunk 1]
```diff
@@ -6,3 +6,5 @@ version: [[ .Version ]]
 appVersion: [[ .AppVersion ]]
 annotations:
   config.giantswarm.io/version: 1.x.x
+annotations:
```
Comment (njuettner) [inline helm/aws-operator/Chart.yaml:9]: wait isn't that duplicate with line 7 😄

**Review by njuettner — CHANGES_REQUESTED** [review]

File: helm/aws-operator/Chart.yaml (hunk 1 above)
Comment (rossf7) [reply helm/aws-operator/Chart.yaml]: @njuettner Damn YAML overload this afternoon. 😅 

Thanks thats fixed and nice CI caught it too. PTAL

**Review by njuettner — APPROVED** [review]
🚀

File: helm/aws-operator/Chart.yaml (hunk 1 above)
Comment (njuettner) [reply helm/aws-operator/Chart.yaml]: Thanks for adding it 👍🏻

**Pull request merged**